GVRf/Framework/src/org/gearvrf/GVRCameraRig.java

```diff
@@ -156,6 +159,8 @@ public float getFloat(String key) {
      *            The {@code float} value to map.
      */
     public void setFloat(String key, float value) {
+        checkKeyNotNullOrEmpty(key);
+        checkNotNaN(value);
```

Comment:
Is infinity a valid value, here? Suggest

    void checkNormalFloat(String name, float value) throws // ...